src/library/Uno.Themes.WinUI.Markup/Theme/TextBlock.cs

```diff
@@ -1,72 +1,81 @@
-﻿using Microsoft.UI.Xaml;
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Media;
 using Uno.Extensions.Markup;
 using Uno.Extensions.Markup.Internals;
 
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
 namespace Uno.Themes.Markup
 {
 	public static partial class Theme
 	{
-		public static class TextBlock
+		public static partial class TextBlock
 		{
-			public static class Styles
+			public static partial class Resources
+			{
+			}
+
+			public static partial class Styles
 			{
-				[ResourceKeyDefinition(typeof(Style), "DisplayLarge", TargetType = typeof(TextBlock))]
-				public static StaticResourceKey<Style> DisplayLarge => new("DisplayLarge");
+				[ResourceKeyDefinition(typeof(Style), "MaterialDisplayLarge", TargetType = typeof(global::Microsoft.UI.Xaml.Controls.TextBlock))]
```

Comment:
Since this is a Themes Markup lib and not a Material specific markup lib, we might need to think about using the semantic design-agnostic style names for all the controls instead